web/angular-wallet/src/app/layout/components/burst-recipient-input/burst-recipient-input.component.ts

```diff
@@ -4,6 +4,9 @@ import {AccountService} from '../../../setup/account/account.service';
 import jsQR from 'jsqr';
 import { NotifierService } from 'angular-notifier';
 
+// generate a unique id for 'for', see https://github.com/angular/angular/issues/5145#issuecomment-226129881
+let nextId = 0;
```

Comment:
is it ok, that the Id resets on each restart? Otherwise, we should reach out for something like UUIDs

Reply:
yeah its fine